runtime/protocol/http/common/test/aws/smithy/kotlin/runtime/http/middleware/MutateHeadersTest.kt

```diff
@@ -95,11 +97,11 @@ class MutateHeadersTest {
         // should leave in existing
         assertEquals("qux", call.request.headers["baz"])
 
-        return@runSuspendTest
+        return@runTest
```

Comment:
question

Do we even need these returns?

Reply:
I didn't notice this.  No, removed.